ini/windows/bin/sc.ini

```diff
@@ -66,6 +70,10 @@ LINKCMD=%VCINSTALLDIR%\bin\link.exe
 ; C Runtime libraries
 LIB=%LIB%;"%VCINSTALLDIR%\lib\amd64"
 
+; Platform/UCRT libraries (Windows SDK 10.0)
+LIB=%LIB%;"%UniversalCRTSdkDir%\Lib\%UCRTVersion%\um\x64"
+LIB=%LIB%;"%UniversalCRTSdkDir%\Lib\%UCRTVersion%\ucrt\x64"
```

Comment:
The same happens as with the other LIB settings: it will point to some (probably) not existing directory and will not cause any trouble.
Leaving it in will make the link work if you didn't let the installer patch sc.ini or it failed to detect the folders, but run `vsvarsall x86/x64` before invoking dmd.
On the other hand we can also remove the UCRT settings here because dmd already detects the existence of the environment variables and adds the `ucrt` path (but misses the `um` path so far). I'm not a huge fan of this magic, though, and I'd prefer to remove it from the compiler (including the other magic library paths).